test/unit/api/getVersions_spec.js

```diff
@@ -2,6 +2,7 @@ const { expect } = require('chai')
 const sinon = require('sinon')
 
 const getVersions = require('../../../src/api/getVersions')
+const { version } = require('../../../package.json')
```

Comment:
Neat. Didn't know you could just import `package.json`.

Reply:
you can `require` any json file